phpcs.xml.dist

```diff
@@ -68,6 +68,16 @@
 		<exclude-pattern>includes/compat/Web_Stories_Compatibility.php</exclude-pattern>
 	</rule>
 
+  <!-- Namespacing of `use` statements. -->
+  <rule ref="PSR2.Namespaces.UseDeclaration">
+    <exclude name="PSR2.Namespaces.UseDeclaration.MultipleDeclarations" />
+  </rule>
+  <rule ref="PSR2R.Namespaces.UnusedUseStatement" />
```

Comment:
Can you test whether using `SlevomatCodingStandard.Namespaces.UnusedUses` instead of this causes the same result? Apparently `fig-r/psr2r-sniffer` includes that sniff.

Basically, want to know whether this prevents https://github.com/google/web-stories-wp/issues/7754#issuecomment-849767697

Reply:
Rephrasing: maybe `SlevomatCodingStandard.Namespaces.UnusedUses` better detects class usage in PHPDoc than `PSR2R.Namespaces.UnusedUseStatement` does. That would be nice.